src/Microsoft.AspNetCore.BuildTools.ApiCheck/ApiListingComparer.cs

```diff
@@ -76,6 +76,24 @@ private void CompareMembers(TypeDescriptor type, TypeDescriptor newType, List<Ap
                     }
                 }
             }
+
+            if (type.Kind == TypeKind.Interface && type.Members.Count < newType.Members.Count)
```

Comment:
Is it sufficient to verify counts? For instance if the member was renamed, the count is the same.